test/integration/agent/openflow_test.go

```diff
@@ -510,6 +524,31 @@ func checkConjunctionFlows(t *testing.T, ruleTable uint8, dropTable uint8, allow
 	}
 }
 
+func checkOVSFlowMetrics(t *testing.T, client ofClient.Client) {
+	// Following is the break-up of flow count for individual flow tables.
+	expectedFlowCount := `
+	# HELP antrea_agent_ovs_flow_count [STABLE] Flow count for each OVS flow table. The TableID is used as a label.
+	# TYPE antrea_agent_ovs_flow_count gauge
+	`
+	tableStatus := client.GetFlowTableStatus()
+	totalFlowCount := 0
+	for _, table := range tableStatus {
+		expectedFlowCount = expectedFlowCount +
+			`antrea_agent_ovs_flow_count{table_id="` + strconv.Itoa(int(table.ID)) + `"} ` + strconv.Itoa(int(table.FlowCount)) + "\n"
```

Comment:
would maybe be more readable with `fmt.Sprintf`? what do you think?

Reply:
That is a good suggestion. Done.